rosbag2_cpp/include/rosbag2_cpp/readers/sequential_reader.hpp

```diff
@@ -101,6 +102,13 @@ class ROSBAG2_CPP_PUBLIC SequentialReader
   */
   virtual std::string get_current_uri() const;
 
+  /**
+   * Add callbacks for events that may occur during bag reading.
+   *
+   * \param callbacks the structure containing the callback to add for each event.
+   */
+  void add_event_callbacks(bag_events::ReaderEventCallbacks & callbacks) override;
```

Comment:
IMO. it will be much safer if `callbacks` parameter will be a const references.

Reply:
Done in db2fc1d.